dashboard/client/package.json

```diff
@@ -126,5 +127,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8265"
+  "proxy": "http://100.82.253.32:8090"
```

Comment:
intentional?

Reply:
Sorry, that's a mistake commit. delete now